File: java/src/main/java/com/iteso/nintendo/attacks/Ember.java
package com.iteso.nintendo.attacks;

public class Ember implements Attack {
    static int damage;
    static String name = "Ember";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/Ember.py
class Ember:
    def __init__(self):
        self.damage = None
        self.name = 'Ember'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
